easystage4.html

```diff
@@ -0,0 +1,59 @@
+<!Doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>repl.it</title>
+    <link href="style.css" rel="stylesheet" type="text/css" />
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+  </head>
+  <body>
+    <div class = 'container-fluid'>
+      <br>
+      <br>
+      <!----Change: description---->
+      <h1 class = 'white' style='font-size: 50px;'>Greek</h1>
+      <p class = 'white' style='font-size: 20px;'>Focus on the centre</p>
+      <p class = 'white'style='font-size: 16px;'><a href = 'https://i.imgur.com/sNYEuDf.png'>https://i.imgur.com/sNYEuDf.png</a></p>
+      <p class = 'white' style='font-size: 15px;'>Origin: Corinth, Greece</p>
+      <!-------------------------->
+      <br>
+      <br>
+      <div class = 'key-input'>
+        <form method = 'POST' onsubmit = 'submitAnswer()'>
+          <div class = 'key-input-group d-block text-center'>
+            <!----Change---->
+            <div class = 'key-input-label d-block text-center'>Key 4
+            </div>
+            <!------------->
+            <br>
+            <div class = 'key-input-answer d-block text-center px-2 mb-3'>
+              <!----Change: number of boxes---->
+              <div class = 'word d-inline-block'>
+                <input name = 'key_input[]' maxlength ='1' class = 'letter'>
+                <input name = 'key_input[]' maxlength ='1' class = 'letter'>
+                <input name = 'key_input[]' maxlength ='1' class = 'letter'>
+                <input name = 'key_input[]' maxlength ='1' class = 'letter'>
+                <input name = 'key_input[]' maxlength ='1' class = 'letter'>
+                <input name = 'key_input[]' maxlength ='1' class = 'letter'>
+                <input name = 'key_input[]' maxlength ='1' class = 'letter'>
+              <!---------------------------->
+              </div>
+            </div>
+          </div>
+          <div class="text-center">
+            <button type="submit" id='submit' class='submit'>SUBMIT</button>
+          </div>
+        </form>
+      </div>
+      <br>
+      <br>
+      <br>
+      <!----Change: number of hints---->
+      <div class='text-center'>
+        <button type='button' class='hint' id='submit'>Reveal Hint</button>
```

Comment:
Why is the ID of this element also `submit` when the ID of the element on line 45 is also `submit`? ID's are not meant to be replicated across different elements. please take note. 

Also `Reveal Hint` button doesn't work